### PR TITLE
[OPPRO-299] Support acos(), asin(), atan(), ceiling(), cos(), cosh(), degrees(), log10() functions in Gluten

### PR DESCRIPTION
**Pull request by zhixingheyi-tian**
## What changes were proposed in this pull request?

Support acos(), asin(), atan(), ceiling(), cos(), cosh(), degrees(), log10() functions in Gluten

## How was this patch tested?

UT added

### CODE REVIEW

File: jvm/src/main/scala/io/glutenproject/expression/ConverterUtils.scala [hunk 1]
```diff
@@ -458,6 +458,23 @@ object ConverterUtils extends Logging {
...
+
+  // PrestoSQL Math fucctions of Velox
+  final val ACOS = "acos"
+  final val ASIN = "asin"
+  final val ATAN = "atan"
+  final val ATAN2 = "atan2"
+  final val COS = "cos"
+  final val COSH = "cosh"
+  final val DEGREES = "degrees"
+  final val LOG10 = "log10"
```
Comment (lviiii) [inline jvm/src/main/scala/io/glutenproject/expression/ConverterUtils.scala:473]: This is a duplicated pr for these functions, please re-check. #520

Comment (zhixingheyi-tian) [reply jvm/src/main/scala/io/glutenproject/expression/ConverterUtils.scala:473]: Seen.
Only one day apart.
Will review each other.

**Review by zzcclp — APPROVED** [review]
LGTM

File: jvm/src/main/scala/io/glutenproject/expression/ConverterUtils.scala (hunk 1 above)
Comment (lviiii) [reply jvm/src/main/scala/io/glutenproject/expression/ConverterUtils.scala]: LGTM

**Comment by lviiii** [comment]
@zhixingheyi-tian Please rebase and fix the conflicts.

**Pull request merged**